scripts/lldpshow

```diff
@@ -25,30 +25,72 @@ import re
 import sys
 import xml.etree.ElementTree as ET
 from tabulate import tabulate
+import argparse
+import sonic_device_util
+from swsssdk import ConfigDBConnector, SonicDBConfig
+BACKEND_NPU_INTERFACE_NAME_PREFIX = 'Ethernet-BP'
+
+LLDP_INTERFACE_LIST_IN_HOST_NAMESPACE ''
+LLDP_INSTANCE_IN_HOST_NAMESPACE ''
+LLDP_DEFAULT_INTERFACE_LIST_IN_NPU_NAMESPACE ''
```

Comment:
Why is NPU namespace also an empty string?

Reply:
@jleveque This is just initial/default value to start with. The Interface list will get appended in the loop for asic namespace. We start with '' and then append to it.

Reply:
I'm confused as to why the host namespace is the "default" in the NPU namespace. Maybe the naming is unclear?

Reply:
@jleveque  for LLDP we have instance in both host (for eth0) and per asic (for physical ports) namespace.

For host namespace using macro LLDP_INSTANCE_IN_HOST_NAMESPACE and LLDP_INTERFACE_LIST_IN_HOST_NAMESPACE.

For per asic namespace using macro LLDP_DEFAULT_INTERFACE_LIST_IN_ASIC_NAMESPACE which is just init value and the interface list gets populated in the loop.

Reply:
@jleveque did I clarify your above concern ? let me know

Reply:
I guess I'm still confused. So the empty string here does not denote the namespace, but rather denotes an interface name? Why do we need to populate an empty interface name here?

Reply:
@jleveque Yes, We have empty string for both namespace and interface-list for host. 

Empty interface-list is to keep logic generic since for host lldp docker as we do not filter of any interface and the command will be just 'lldpctl' + <Empty string>.
In case user has passed specific interface (-p option)  command will be 'lldpctl' + < Interface-Num>.

For asic namespace we filter out Backplane interface (Ethernet-BP) and command will be 'lldpctl' + <Front Interface list> 
or in case user has passed interface (-p option) command will be 'lldpctl' + < Interface-Num>.

Reply:
I see. Thanks for the explanation. I guess it might just be the similarity of the constant names that is throwing me off here. I'd also like to get Pavel's input here, who is more of a SME on LLDP.

Reply:
@jleveque can you please approve this. pavel has approved it.